README.md

```diff
@@ -3,7 +3,60 @@
 The `metabase-materialize-driver` allows [Metabase](https://github.com/metabase/metabase) 
 to connect to an instance of Materialize.
 
-## How to use the driver
+![Choose Materialize from database dropdown](images/materialize-metabase.png)
+
+## To Use the Driver
+
+To use the `metabase-materialize-driver`, put a copy of the driver's
+.jar file into the `/plugins` directory of your Metabase instance. Metabase
+will register the driver automatically! (For deployment-specific details, 
+please consult the following sections.)
+
+Once the Materialize driver is registered, use the following information
+to connect: 
+
+| Field             | Value     |
+| ----------------- |:---------:|
+| Host              | localhost |
+| Port              | 6875      |
+| Database username | default   |
+| Database password | default   |
+
+
+### With Docker
+
+To generate a Metabase Docker image containing the Materialize driver,
+use [our Dockerfile](Dockerfile):
+```shell script
```

Comment:
this should probably use `Dockerfile` syntax, but the other option is just omitting this and leaving the link.